UnitTests/expected/Storyboards-osx-Message-Defaults.swift.out

```diff
@@ -46,22 +46,22 @@ struct StoryboardScene {
 
     case MessageDetailsScene = "MessageDetails"
     static func instantiateMessageDetails() -> NSViewController {
-      return StoryboardScene.Message_Osx.MessageDetailsScene.controller()
+      return StoryboardScene.Message_Osx.MessageDetailsScene.controller() as! NSViewController
```

Comment:
Ah, indeed the warning was not about trailing whitespaces in empty lines, but about trailing newlines at end of file, I misread it sorry.

That's probably because of the way I parse `.swift.out` files with SwiftLint. Since SwiftLint is only parsing files with extension `*.swift` given a directory, I had to cheat a little in the `scripts/swiftlint-templates.sh` script and instead use something like `cat *.swift.out | swiftlint lint --use-stdin`. That's probably because of that `cat`+pipe that SwiftLint considers there is a missing line at end of that input, probably because `cat` eats that trailing line?
Strange that I don't have that warning on my end, will try to investigate more.

---

In the meantime, [I added the `force_unwrapped` opt-in rule in `.swiftlint.yml` on `master`, which revealed that my templates for storyboards still had a `!` somewhere in their code](https://github.com/AliSoftware/SwiftGen/commit/9326ef4376086fd716d45c765c203621b39d674c). So I fixed that since then but you might want to pick or rebase on that change to ensure your OS X storyboard templates don't have those `!` hidden somewhere as well.